packages/documentation/src/shared/package-tag.mdx

```diff
@@ -0,0 +1,3 @@
+<div className={'tag tag-sm mb-24 tag-' + (props.meta.tags[0] == 'package:HTML' ? "" : "yellow")}>
```

Comment:
Here, if the tag is "package:HTML" you will end up with an unwanted `tag-` class. You can see it here by inspecting the package tag here: https://preview-5984--swisspost-design-system-next.netlify.app/?path=/docs/bec68e8b-445e-4760-8bd7-1b9970206d8d--docs